# Stetige vs. Diskrete Verteilungen in R

## Die Poisson-Verteilung

Zählt die ANzahl von Ereignissen.

R - Random Sample

In [1]:
rpois(10, lambda = 8) 

[1] 10  3  6  8 14  7  9 13  8  9

D - Distribution

In [2]:
dpois(10, lambda = 8)

[1] 0.09926153

$\Rightarrow$ die WKeit, dass genau 10 Ereignisse auftreten, wenn die zugrundeliegende Rate 8 ist, beträgt 0.09926

d.h.  D ist im diskreten Fall die Wahrscheinlichkeitsfuntion.

P - Probability

In [3]:
ppois(10, lambda = 8)

[1] 0.8158858

Hier kommt immer die Warhscinelichkeit, dass ein Wert $\leq$ x (in diesem Fall 10) angenommen wird.

Q - Quantile

In [4]:
qpois(0.1, lambda = 8)

[1] 5

d.h. dass 90% aller Werte größer und 10% aller Werte kleiner als 5 sind. 

## Exponentialverteilung.

Ist eine stetige Verteilung. Gibt das Wahrscheinlichkeitsprofil an für die Zeit, die man Warten muss, bis ein bestimmtes Ereignis eintritt. 

In [5]:
rexp(10, 1/20)

[1] 35.098101 56.165508  2.431676 42.779297 70.344090  2.149863  7.599471
 [8] 17.559432  6.826532 20.035134

Auch bei der Expoentialverteilung liegt ein Entstehungsprozess zugrunde der eine bestimmte Rate hat. 

In [6]:
rexp(10, 20)

[1] 0.0819187197 0.0453858584 0.0323390332 0.0093759640 0.0579572385
 [6] 0.0835630866 0.0312558598 0.0004108542 0.0243658172 0.0051575746

D - Distributiokn

In [7]:
dexp(10, 1/20)

[1] 0.03032653

Das ist keine Wahrscheinlichkeit! 

P - Probability

In [8]:
pexp(10, 1/20)

[1] 0.3934693

Das ist eine Wahrscheinlichkeit, nämlich die dass es weniger als 10 s dauert, bis das erste Ereignise eintritt. 

Hier hat R schon integriert. 